### ISSUE TITLE
Voting interrupting the show even when not checked

### ISSUE THREAD

**Issue by ioscanner** [Remote-Falcon/remote-falcon-issue-tracker#83]
I thought this use to work in past years.  When in voting mode there was an option to not interrupt the show there still is an option but it doesn't seem to do anything.  Now it does after a vote for a song the next song is the voted song.  I would like it to have the option again to not Play the voting songs at all or to have it play the highest voted song for that round to play it again at the end of the playlist and then zero them and start over again.

For now the most import is to not interrupt as I can't leave voting on or they will never get thru the playlist.

**Comment by whitesoup12** [comment]
The option to toggle whether to interrupt the scheduled show is in the plugin. As for resetting votes back to 0, you can find that in the Voting settings (Reset Votes).

**Comment by ioscanner** [comment]
Yes, as I said I tried that and it doesn't work. It use to work. When a person votes it plays their song next not at the end of the playlist. I would rather don't play it just do voting.

**Comment by whitesoup12** [comment]
Interrupt will not wait until the end of a playlist, just to the end of the sequence. Closing as a non-issue.
